docs/modules/usage/llms/llms.md

```diff
@@ -26,8 +26,10 @@ The following environment variables might be necessary for some LLMs/providers:
 
 We have a few guides for running OpenHands with specific model providers:
 
+- [OpenAI](llms/openai-llms)
 - [ollama](llms/local-llms)
 - [Azure](llms/azure-llms)
+- [Google](llms/google-llms)
```

Comment:
Should we do this alphabetically?

Reply:
Or maybe based on popularity instead? 🤔

- OpenAI
- Azure
- Google
- ollama

Reply:
Or maybe based on popularity instead? 🤔

OpenAI
Azure
Google
ollama

Reply:
So what if I said Google was more popular that Azure? 😏

Reply:
🔫

Reply:
Then you would have to open a new PR to change the order and hope I approve it

Reply:
Typing about it took more time than to do it. Already approved, was just a nit. 🤗